# Powerful Gene Clusters Could Revolutionize Chickpea Farming in Salt Sick Indian soils 

![9a](https://user-images.githubusercontent.com/85151795/188297247-d66cf454-3af4-4327-9337-8bf2a0ba2aa6.jpg)

## Introduction
Global climate change and India’s mammoth population growth are the major issues leading to drastic environmental stress on agriculture and food production. Adverse effect of climate change has started showing manoeuvred and intensified climatic aberrations like, paucity of water, escalation in sea levels, heat waves, rapid floods etc. Frequent water scarcity nowadays forcing restriction on the use of fresh water for irrigating crops. In days to come agriculture will be forced to thrive with waste effluent and contaminated waters. These water have highly toxic ions manifolding salinity. Apart from this India has about about 147 million ha of degraded land out of which about 6.74 million ha is salt-affected. Studies suggest 10% additional area is getting salinized annually and by 2050 about 50% of the arable land would be salt affected. Major problem caused by high salinity is difficulty of plant uptake of water to fulfil its transpiration demand along with, nutrients deficiencies. The population growth projection says that from 2030, India have to feed 1.43 billion people, to achieve this target agriculture has to be expanded in non-conventional marginalised area via restoring the degraded and salt sick lands. There are several approaches to combat and manage salinity like, soil reclamation, agronomic practices, chemical amendments and tolerant varieties. Among these breeding tolerant cultivars are the most preferred strategy.

![9b](https://user-images.githubusercontent.com/85151795/188297253-eef30dce-0159-4eec-99e7-95f296abcfd8.jpg)

Deployment of genes/ QTLs (cluster of many genes) for crop improvement is now a trending prodigyin agricultural science. Locating useful bunch of genes (QTLs) in genome and incorporating them to work and serve our purpose is the most cost effective and preferred strategy in plant breeding. A multi-disciplinary team of dedicated researchers from the field of plant breeding, molecular biology, plant physiology, soil science and bioinformatics lead by Chickpea Breeder Dr Bharadwaj from Division of Genetics, Indian Agricultural Research Institute (IARI), Pusa Institute. India’s National Institute for Agricultural Research, Education and Extension under the administrative control of ICAR situated at New Delhi are focused on discovering the sustainable solution to beat the increasing salt affected areas via breeding tolerant chickpea cultivar that can be cultivated in salty areas with salty water 
Chickpea (Cicer arietinum L.) is one of the most important grain legume crops with worldwide production of 17.2 million tons, including 11.38 million tons in India (FAO, 2018). Chickpea provides a large fraction of protein and essential amino acids for populations in arid and sub-arid regions of the world. However, chickpea is a salt-sensitive legume, showing reduced growth parameters and excessive salinity can reduces annual chickpea yields by 8–10 % globally. Salinity tolerance is governed by traits involved in complex physiological and biochemical responses by inducing multiple genes under stress. Excess salinity induces ionic imbalance via shoot-independent and shoot-dependent ion stresses, affecting plant growth and development. A reduction in external osmotic potential restricts water uptake, which affects plant growth and reduces biomass yield. Ion toxicity mainly affects the Na+ signal transduction pathway, which is critical for detoxification and balancing cellular ion homeostasis. The K+ ion is essential for important metabolic processes in the cytoplasm, including protein synthesis, enzymatic reactions, and ribosome functions. Thus, the Na+/K+ ratio is an important determinant of salt stress tolerance; maintaining cellular Na+/K+ is a basic feature of tolerant plants under salt stress. Our current study had identified genes associated with salt stress and physio-biochemical factors responding to salinity, which will largely assist the development of improved chickpea varieties with enhanced salinity tolerance.  
Gene clusters or QTLs (Quantitative Trait Loci) can be analogue with molecular engines, cluster of more number of genes/ QTLs means more power. The salinity tolerance in plants is not controlled by single gene it is a quantitative trait, various genes all together influence this trait. The collective consortium of all salt responsive genes together if incorporated in varieties lacking them via molecular or conventional breeding strategy can make these varieties tolerant and capable to withstand salinity stress. The advanced breeding material obtained via accruing more potent molecular engine will enhance the thrust and performance of chickpea plant under saline water/soil.  
## Signalling under stress 
As the salinity becomes unbearable plant suffers from oxidative, osmotic and ionic stress. As the stress goes severe, the exposed plants starts sending signals to activate its protective machinery via calcium dependent protein kinases pathways. The tolerant genotypes activate its defence system more proficiently by activating transporter genes which maintain the ionic balance by extrusion of the toxic Na+ to maintain proper Na+/K+ ratio along with cellular redox balance and mitigates damage caused by reactive oxygen species. The plant defence system also controls the hormonal signalling network which alters physiological responses under salinity and affects cell differentiation, cell elongation and inducing genes for osmoprotectant and stomatal closer aiding plant to cope up with elevated stress. Genes affecting these hormonal pathways have been elucidated, which are crucial to understand the roles of these endogenous molecules regulating several plant developmental processes in acclimating chickpea adaptation to salt stress (Kumar et al., 2020). 
## Map of Genes to facilitate rapid breeding for salt tolerance in chickpea 
For locating genes/QTLs that could enable plants withstand salt stress, chickpea population derived from parents, ICCV 10 (salt tolerant) x DCP 92-3 (salt sensitive) were used (Fig. 1). Extensive characterization of chickpea mapping population under high salt conditions in micro-plot and pot experiments for two consecutive years. The map of QTLs/ Gene clusters imparting salinity tolerance in chickpea was constructed. High density map comprising 1856 SNP markers spanning a distance of 1106.3 cM across eight chickpea chromosomes led to identification of 28 major and minor effect QTLs explaining up to 28.40% of the phenotypic variance (Fig 2) (Kumar et al., 2020; Soren et al., 2020; Kumar et al., 2021c). These genomic clusters harbour the genes which provides tolerance against high salt conditions via regulating traits like, relative water, chlorophyll and proline contents along with reducing electrolyte leakage and making equilibrium in shoot ion distribution. This map will assist in fast forward breeding by specifically transferring tolerant cluster of genes to elite chickpea cultivars in short period of time through molecular breeding skills. Overall mapping and molecular analysis will reinforce the understanding of molecular mechanics underlying metabolites, phyto-hormonal and gene regulatory pathways under stress. The results have also led to the identification of salt responsive candidate gene based SSR marker (cg-SSR) and have been successfully validated. This will serve as an amazing tool to distinguish salt susceptible and salt tolerant chickpea genotypes at molecular level. These results also highlighted the understanding of complex molecular mechanisms and genetics underlying salinity tolerance in chickpea. The gene harbouring in the clusters can be effectively transferred into elite chickpea varieties to make them endure salt stress. With the advent of advances in molecular breeding these processes are precise and fast as compared to conventional breeding approaches. 

![9c](https://user-images.githubusercontent.com/85151795/188297266-402a5448-86fc-4d71-8c4a-5fe2c0c711f4.png)

Characterization of chickpea mapping population and parents under elevated salt stress 

## Molecular Mechanics underlying salinity tolerance
To understand the complex molecular mechanisms underlying salinity tolerance. We took the help of comparative transcriptome analysis data of tolerant (ICCV 10, JG 11) and sensitive (DCP 92-3, Pusa 256) chickpea genotypes in control and salt-stressed environments. A total of 530 million reads were generated from root samples of four genotypes using Illumina HiSeq-2500. A total of 21,698 differentially expressed genes (DEGs) were identified, of which 11,456 and 10,242 were up- and down-regulated, respectively, in comparative analysis. These DEGs were associated with crucial metabolic pathways, including hormone signalling, photosynthesis, lipid and carbohydrate metabolism, and cell wall biogenesis. Gene ontology (GO) examination revealed an enrichment of transcripts involved in salinity response. A total of 4257 differentially expressed GO terms were categorized into 64 functional groups; of which, GO terms like, integral component of membrane, organelle, and cellular anatomical entity were highly represented in tolerant genotypes under salt stress. Significant up-regulation of transcripts encoding potassium transporter family HAK/KUP proteins, MIP/aquaporin protein family, NADH dehydrogenase, pectin esterase, and PP2C family proteins occurred under salt stress. The tolerant lines (ICCV 10 and JG 11) engaged highly efficient machinery in response to elevated salt stress, especially for signal transduction, transport and influx of K+ ions, and osmotic homeostasis (Kumar et al., 2021a; Kumar et al., 2021b). 

![9d](https://user-images.githubusercontent.com/85151795/188297273-75754bb7-a0ce-4266-be3d-abbd28ea584f.jpg)

Molecular map of gene clusters for salinity tolerance identified in chickpea 

## Conclusion 
The high-density map, major QTLs, and genomic regions identified in this study can be used in GAB (genomic assisted breeding), to breed of high-yielding salinity-tolerant chickpea varieties in reasonable time frame with less resources. This study have successfully discern the genetics of salinity tolerance, which could ultimately pave the way to harness molecular breeding tactics to elevate salinity tolerance in chickpea breeding programs. 
## References:
- Kumar, N., Soren, K.R., Bharadwaj, C., PR, S.P., Shrivastava, A.K., Pal, M., Roorkiwal, M., Kumar, K., Patil, B.S., Soni, A. and Nimmy, M.S., 2021a. Genome-wide transcriptome analysis and physiological variation modulates gene regulatory networks acclimating salinity tolerance in chickpea. Environmental and Experimental Botany, 187, p.104478. 
- Kumar, N., Bharadwaj, C., Sahu, S., Shiv, A., Shrivastava, A.K., Reddy, S.P.P., Soren, K.R., Patil, B.S., Pal, M., Soni, A. and Roorkiwal, M., 2021b. Genome-wide identification and functional prediction of salt-stress related long non-coding RNAs (lncRNAs) in chickpea (Cicer arietinum L.). Physiology and Molecular Biology of Plants, 27(11), pp.2605-2619. 
- Kumar, N., Bharadwaj, C., Soren, K.R., Meena, M.C., Priya, P.S., Joshi, N., Soni, A., Patil, B.S., Pal, M., Roorkiwal, M. and Varshney, R.K., 2021c. Response of shoot ionic (Na+/K+) distribution on yield and agro-physiological traits in chickpea (Cicer arietinum L.) recombinant inbred lines population under salinity. INDIAN JOURNAL OF GENETICS AND PLANT BREEDING, 81(04), pp.92-95. 
- Soren, K.R., Madugula, P., Kumar, N., Barmukh, R., Sengar, M.S., Bharadwaj, C., Sharma, P.C., Singh, S., Bhandari, A., Singh, J. and Sanwal, S.K., 2020. Genetic dissection and identification of candidate genes for salinity tolerance using Axiom® CicerSNP array in chickpea. International journal of molecular sciences, 21(14), p.5058. 
- Kumar, N., Bharadwaj, C., Soni, A., Sachdeva, Supriya., Yadav, M.C., Pal, Madan Soren, K.R., Meena, M.C., Roorkiwal, Manish., Varshney, R.K. and Rana, Maneet., 2020. Physio-morphological and molecular analysis for salt tolerance in chickpea (Cicer arietinum). Indian Journal of Agricultural Sciences, 90(4), pp.132-136.

### ABOUT THE AUTHOR
### Neeraj Kumar and Chellapilla Bharadwaj
Chickpea Breeding & Molecular Breeding Programme, Division of Genetics, ICAR- Indian Agricultural Research Institute, Pusa, New Delhi 110 012. 
*Correspondence: Chellapilla Bharadwaj
### Email
[mailto:chbharadwaj@yahoo.co.in](mailto:chbharadwaj@yahoo.co.in)
